XML/TestCases/Other.xml

```diff
@@ -51,4 +51,14 @@
         <Area></Area>
         <Tags>gcov</Tags>
     </test>
+    <test>
+        <testName>LONG-PERF-STRESS-TEST</testName>
+        <testScript>DEPLOY-LONG-TERM-VM.ps1</testScript>
+        <setupType>LPST</setupType>
+        <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\prepare_ltpt_vm.sh,.\Testscripts\Linux\CreateRaid.sh</files>
+        <Platform>Azure</Platform>
+        <Category></Category>
```

Comment:
Category & Area are missing

Reply:
no need for these, this is a specialized scenario and we don't want it triggered by Category/Area